custom/icds_reports/templates/icds_reports/dashboard.html

```diff
@@ -132,50 +133,59 @@ <h2 class="dashboard-title">
                               <nav class="navbar-menus fullmenu" id="hq-full-menu" role="navigation">
                                   <div class="nav-settings-bar pull-right">
                                       <ul class="nav navbar-nav" role="menu">
-                                          <li class="dropdown">
+                                          <li class="nav-menu-icon">
                                               <span class="pointer">
                                                   <a href="{% static 'media/Dashboard_Training_Manual.pdf' %}" download>
                                                       <i class="fa fa-question-circle nav-main-icon fa-2x light-grey"
                                                           uib-popover-html="'Download user manual'"
                                                           popover-placement="left"
-                                                          popover-trigger="'mouseenter'" style="margin-top: 11px;"></i>
+                                                          popover-trigger="'mouseenter'" ></i>
                                                   </a>
                                               </span>
                                           </li>
-                                          <li class="dropdown" ng-hide="$location.path().indexOf('download') !== -1">
+                                      {% if show_mobile_apk %}
```

Comment:
for my own knowledge. Is there a reason it has to use its separate feature flag and cant use this general one https://www.icds-cas.gov.in/hq/flags/edit/features_in_dashboard_icds_reports/

Reply:
I suspect that we'll want to "release" it to everyone at some unknown point in the future, and if it's a `DynamicallyPredictablyRandomToggle` then we can do that without a deploy.